### DISCUSSION TITLE
Services available to help you with your Hyperledger project or lab

### DISCUSSION THREAD

**Discussion by davidwboswell** [hyperledger/governance#88] [Team Posts]
I wanted to share with people running a project or lab that we've posted a document on the wiki that details what services are available to you.  That is at:

https://wiki.hyperledger.org/display/HYP/Project+and+Lab+Services

Most of the items there have been available in the past, although you may not be aware of what you can make use of since it wasn't documented in one place like this.

Note that there is one time specific service you may be interested in -- we're gathering project proposals for our annual Mentorship program.  If you'd like to be matched with a mentee to help with your project or lab, project proposals are due by March 9.

https://wiki.hyperledger.org/display/INTERN/Hyperledger+Mentorship+Program

If you're interested in any of the services, feel free to reach out to me or to the contact email on the page.  And if there are other services you would be interested in that aren't on the doc, we welcome your feedback and can talk with you about what's possible to add.

Thanks,
David

**Comment by kuvaldini** [comment]
Hello. You can suggest to add this information to iroha official documentation at https://iroha.readthedocs.org . Via PR or issue to https://github.com/hyperledger/iroha
